sshd/issue_invite.sh

```diff
@@ -28,36 +28,37 @@ done
 shift $((OPTIND-1))
 
 TMP=`mktemp -d`
+npm install jsurl &>/dev/null
 
 # Either read the supplied invite code or generate a new one.
 if [ -n "$INVITE_CODE" ]
 then
-  INVITE=`echo -n $INVITE_CODE|base64 -d`
+    INVITE=`echo -n $INVITE_CODE|base64 -d`
```

Comment:
Ah...whitespace changes. Let's leave this for a followup? Makes it harder to see which changes are related to the purpose of this pull request.